atomic_reactor/plugins/flatpak_create_oci.py

```diff
@@ -85,9 +85,10 @@ def build_flatpak_image(self, source, build_dir: BuildDir) -> Dict[str, Any]:
 
         cmd = ['skopeo', 'copy', 'oci:{path}:{ref_name}'.format(**metadata), '--format=v2s2',
                'docker-archive:{}'.format(str(build_dir.exported_squashed_image))]
+        cleanup_cmd = ['rm', str(build_dir.exported_squashed_image)]
```

Comment:
Hmm, just noticed that we convert the `Path` to string here but not above. Is it needed or not?